draft-ietf-rats-corim.md

```diff
@@ -166,11 +166,19 @@ This specification describes the CoRIM format and documents how a Verifier shoul
 A Verifier needs to reconcile its various inputs, with CoRIM being one of them.
 In addition to the external CoRIM documents, the Verifier is expected to create an internal representation for each input and map each external representation to an internal one.
 By using the internal representation, the Verifier processes inputs as if they are part of a conversation, keeping track of who said what.
-The origin of the inputs is tracked as *authority*.
-The authority for the Claims in a CoRIM is the CoRIM issuer.
 
 Effectively, Attesters, Reference Value Providers, Endorsers, Verifier Owners, Relying Parties, and even the Verifier potentially all contribute to the conversation.
-Each producer of corresponding RATS Conceptual Messages can assert Claims about an Attester's actual or allowed state.
+These individual principals of the conversation may have their identity rooted back to some *authority* farther up the certificate chain than the signing key itself, such as a digital certificate authority intermediate or root key.
+The origin of contributions/inputs to the conversation are tracked at the granularity of authority, so CoRIM authors can predicate their reference measurements or endorsements based on existing properties rooted from the same authority.
+
+CoRIM inputs to the conversation include measurements within a specific Environment.
+A measurement never appears outside the scope of an associated Environment, so the Environment-Claim Tuple (ECT) is a data structure used to encapsulate all scope context for a given set of measurements.
```

Comment:
Maybe: 
```suggestion
A specific set of measuremenst always originates from a single identifiable and associated Environment. Hence. an Environment-Claim Tuple (ECT) is a data structure used to encapsulate that scope context for a given set of measurements.
```

Reply:
This suggestion is saying much more than I'm saying, since I don't insist that each specific set of measurements has a single environment. Indeed, sets of measurements don't necessarily have a globally unique identifier for distinguishing them across individual measurements. I can have the same card on two different sockets and have the same measurement set, but a different environment due to my assignment of card to group.

Perhaps "specific set" is too ambiguous. Let's go with "a specific occurrence of a set", since the occurrence refers to where it is in the CoRIM.